package.json

```diff
@@ -79,5 +80,14 @@
       "react",
       "stage-2"
     ]
+  },
```

Comment:
I believe we deleted the jest configuration, and we also already have the pg-promise dependency.